ludwig/utils/strings_utils.py

```diff
@@ -136,10 +145,19 @@ def _get_sequence_vector(
     return unit_indices_vector
 
 
-def build_sequence_matrix(sequences, inverse_vocabulary, format, length_limit,
-                          padding_symbol, padding='right',
-                          lowercase=True):
-    tokenizer = get_from_registry(format, tokenizer_registry)()
+def build_sequence_matrix(
+    sequences,
+    inverse_vocabulary,
+    tokenizer,
+    length_limit,
+    padding_symbol,
+    padding='right',
+    lowercase=True,
+    tokenizer_vocab_file=None,
+):
+    tokenizer = get_from_registry(tokenizer, tokenizer_registry)(
```

Comment:
here we are overriding the name tokenizer by reassigning, maybe tokenizer_type would be better as a name in the signature to avoid the clash